c_binding/lsm_local_disk.c

```diff
@@ -751,3 +784,182 @@ int lsm_local_disk_vpd83_get(const char *disk_path, char **vpd83,
 
     return rc;
 }
+
+/*
+ * Assuming vpd_0_len > 3 and vpd_0_data is not NULL, caller should make sure
+ * that.
+ */
```

Comment:
caller should make sure of that.  

Or just re-word to: 

Preconditions: vpd_0_len > 3,  vpd_0_data is not NULL and vpd_0_len > ?